README.md

```diff
@@ -238,71 +269,69 @@ code changes and content changes, they will be caught early on. The Storyblok
 editor automatically passes cache-busting query string parameters (as in
 `?_storyblok=<timestamp>`) to the preview URL, so it should be okay to use a
 constant URL base, such as
-quansight-consulting-git-develop-quansight.vercel.app, rather than the always
+`quansight-consulting-git-develop-quansight.vercel.app`, rather than the always
 changing SHA-based Vercel preview URLs
-(quansight-consulting-<SHA>-quansight.vercel.app).
+(`quansight-consulting-<SHA>-quansight.vercel.app`).
 
 There is a fair amount of code in the codebase that integrates with Storyblok.
 For example, there is a bridge that syncs the site with changes coming from the
 Storyblok editor.
 
-There is also some middleware that checks if the HTTP referrer is
-`https://app.storyblok.com/`. In that case, the code assumes that the end user
+Some middleware checks if the HTTP referrer is
+<https://app.storyblok.com/>. In that case, the code assumes that the end user
 is coming to the site from within the Storyblok iframe. So it puts the user in
 Next.js preview mode. When the user is in preview mode, the website shows a
 yellow banner at the top of the page, letting the user know that they are seeing
 draft content on the site.
 
 ### Vercel
 
-Each website corresponds to a separate project in Vercel, quansight-consulting
-and quansight-labs, respectively. However, because both projects are mapped to
+Each website corresponds to a separate project in Vercel, `quansight-consulting`, and `quansight-labs`, respectively. However, because both projects are mapped to
```

Comment:
Reflow

```suggestion
Each website corresponds to a separate project in Vercel, `quansight-consulting`,
and `quansight-labs`, respectively. However, because both projects are mapped to
```